UnitTests/Charcoal/Selection/FilterSelectionStoreTests.swift

```diff
@@ -200,32 +200,22 @@ final class FilterSelectionStoreTests: XCTestCase {
         store.setValue(10, for: lowValueFilter)
         store.removeValues(for: highValueFilter)
         XCTAssertTrue((store.titles(for: filter).count == 1))
-        XCTAssertFalse(store.titles(for: filter)[0].value.isEmpty)
-        XCTAssertFalse(store.titles(for: filter)[0].accessibilityLabel.isEmpty)
-
-//        XCTAssertEqual(store.titles(for: filter), [
-//            SelectionTitle(value: "Fra 10 kr", accessibilityLabel: "Fra 10 kroner"),
-//        ])
+        XCTAssertTrue(store.titles(for: filter)[0].value.contains("10"))
+        XCTAssertTrue(store.titles(for: filter)[0].accessibilityLabel.contains("10"))
 
         store.removeValues(for: lowValueFilter)
         store.setValue(100, for: highValueFilter)
         XCTAssertTrue((store.titles(for: filter).count == 1))
-        XCTAssertFalse(store.titles(for: filter)[0].value.isEmpty)
-        XCTAssertFalse(store.titles(for: filter)[0].accessibilityLabel.isEmpty)
-
-//        XCTAssertEqual(store.titles(for: filter), [
-//            SelectionTitle(value: "Opptil 100 kr", accessibilityLabel: "Opptil 100 kroner"),
-//        ])
+        XCTAssertTrue(store.titles(for: filter)[0].value.contains("100"))
+        XCTAssertTrue(store.titles(for: filter)[0].accessibilityLabel.contains("100"))
 
         store.setValue(10, for: lowValueFilter)
         store.setValue(100, for: highValueFilter)
         XCTAssertTrue((store.titles(for: filter).count == 1))
-        XCTAssertFalse(store.titles(for: filter)[0].value.isEmpty)
-        XCTAssertFalse(store.titles(for: filter)[0].accessibilityLabel.isEmpty)
-
-//        XCTAssertEqual(store.titles(for: filter), [
-//            SelectionTitle(value: "10 - 100 kr", accessibilityLabel: "Fra 10 Opptil 100 kroner"),
-//        ])
+        XCTAssertTrue(store.titles(for: filter)[0].value.contains("10"))
+        XCTAssertTrue(store.titles(for: filter)[0].value.contains("100"))
+        XCTAssertTrue(store.titles(for: filter)[0].accessibilityLabel.contains("10"))
+        XCTAssertTrue(store.titles(for: filter)[0].accessibilityLabel.contains("100"))
```

Comment:
This gives a false sense of security, since `100` will always match `10`.
Can we either use regex or maybe break up the string, i.e. by spaces, and then check if the array contains the values we expect?

Reply:
Veeery good point, fixed ✅